config/k8sdev.js

```diff
@@ -0,0 +1,56 @@
+var merge = require('webpack-merge')
+var base = require('./index.js')
```

Comment:
We could use './dev.js' as the base, and then the rest of this could just be
```
module.exports = merge(base, {
	server: {
		memcachedServers: 'k8sdev-elasticache.bu9ifv.cfg.usw1.cache.amazonaws.com:11211',
	}
})
```

Reply:
@jmallatt-kiva  I'll make this update.